packages/shared/lib/core/wallet/stores/new-transaction-details.store.ts

```diff
@@ -13,8 +14,7 @@ export function resetNewTokenTransactionDetails(): void {
         type: NewTransactionType.TokenTransfer,
         recipient: undefined,
         rawAmount: undefined,
-        asset: undefined,
-        unit: undefined,
+        tokenMetadata: getBaseToken(),
```

Comment:
i dont think these changes are correct, the transaction details uses `asset` because you can send a token or an NFT, lets not change this to avoid breaking NFT transactions